### ISSUE TITLE
Do file uploads with WTForms

### ISSUE THREAD

**Issue by redshiftzero** [lucyparsons/OpenOversight#5]
To enable users to SAFELY upload certain file formats. These uploads should go to S3 and then a row should be inserted into `officers.raw_images`.

**Comment by r4v5** [comment]
possible inspiration: https://github.com/tornadoweb/tornado/blob/master/demos/s3server/s3server.py

**Comment by freddymartinez9** [comment]
Discussion from our Slack channel:
```
@r4v5  i’m wondering if ‘s3, apply directly to the bucket’ is the right answer here, i keep waffling back and forth on it 
@b_meson
we dont have a high volume of submissions. I’m also thinking maybe S3 since our concern about trolls flooding and wasting out money might be overstated
@akama   
s3 is really handy for stuff like this, it's what I use for another application
@b_meson [13:30]  
i’ll leave it up to Dr Helsby, lead developer of OO to decide. I think S3 might be fine. make it amazon’s problem. POST to s3, then return it. We need to routinely do database and S3 backups anyways, which means we can handle malicious files (i.e. malware) in our export. once I write the python fabric for this, I can execute it in a QubesVM so any malware would be isolated. and just remove any non images
@redshiftzero
basically to do the Qubes convert to trusted PDF for images
```

**Comment by redshiftzero** [comment]
Relevant: https://www.owasp.org/index.php/Unrestricted_File_Upload

**Comment by redshiftzero** [comment]
In progress in branch `image-uploading`

**Comment by redshiftzero** [comment]
This was implemented and merged in #149, closing!
